frontend/components/brand/products/product-card.tsx

```
@@ -0,0 +1,91 @@
+      </CardContent>
+      <CardFooter className="flex-col items-start gap-2 p-4">
+        <div className="flex items-center gap-1">
+          {[...Array(5)].map((_, i) => (
+            <svg
+              key={i}
+              className={cn(
+                "h-4 w-4",
+                i < Math.floor(product.rating) ? "fill-primary" : "fill-muted stroke-muted-foreground",
+              )}
+              viewBox="0 0 24 24"
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)